manim/scene/scene_file_writer.py

```diff
@@ -585,8 +585,8 @@ def clean_cache(self):
             for file_to_delete in oldest_files_to_delete:
                 os.remove(file_to_delete)
             logger.info(
-                f"The partial movie directory is full (> {config['max_files_cached']} files). Therefore, manim has removed {number_files_to_delete} file(s) used by it the longest ago."
-                + "You can change this behaviour by changing max_files_cached in config.",
+                f"The partial movie directory is full (> {config['max_files_cached']} files). Therefore, manim has removed the {number_files_to_delete} least recently used file(s).",
```

Comment:
I think instead of “the {n} least recently used files” it would be better to phrase it as “the {n} oldest files”, it also aligns better with the coding around it. The method docstring also has “…manim the longest ago” that could be reworded to say “ Will clean the cache by removing the oldest partial_movie_files.” But only if you have time as I know it isn’t technically part of this PR.

Reply:
Sounds reasonable. I incorporated both of your suggestions in a new commit.

> But only if you have time as I know it isn’t technically part of this PR.

That part (if I understood it correctly) was no big deal at all. I just hope I don't consume too much of _your_ time with this back-and-forth. :)

Reply:
Not at all, thank you very much for your help, changes look good so will merge soon :)